contracts/DexX.sol

```diff
@@ -0,0 +1,80 @@
+// contracts/GLDToken.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@redstone-finance/evm-connector/contracts/data-services/MainDemoConsumerBase.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract DexX is MainDemoConsumerBase, Ownable {
+
+    bytes32 public constant AVAX_SYMBOL = bytes32("AVAX");
+    uint256 public constant REQUEST_TTL_IN_BLOCKS = 3;
+
+    // We don't need params for the swap requests
+    // because all of them can be extracted from the transaction data
+    event NewSwapRequest();
```

Comment:
Hmm, but it might be easier for keepers to see all the parementers here

Reply:
Could we please change name to NewDataRequest() and add parameters here

Reply:
Done